akka-remote/src/main/scala/akka/remote/serialization/MiscMessageSerializer.scala

```diff
@@ -129,6 +173,34 @@ class MiscMessageSerializer(val system: ExtendedActorSystem) extends SerializerW
     }
   }
 
+  private def deserializeStatusSuccess(bytes: Array[Byte]): Status.Success =
+    Status.Success(deserializePayload(ContainerFormats.Payload.parseFrom(bytes)))
+
+  private def deserializeStatusFailure(bytes: Array[Byte]): Status.Failure =
+    Status.Failure(deserializePayload(ContainerFormats.Payload.parseFrom(bytes)).asInstanceOf[Throwable])
+
+  private def deserializeThrowable(bytes: Array[Byte]): Throwable = {
+    val protoT = ContainerFormats.Throwable.parseFrom(bytes)
+    val t: Throwable =
+      if (protoT.hasCause) {
+        val cause = deserializePayload(protoT.getCause).asInstanceOf[Throwable]
+        system.dynamicAccess.createInstanceFor[Throwable](
+          protoT.getClazzName,
+          List(classOf[String] → protoT.getMessage, classOf[Throwable] → cause)).get
+      } else
+        system.dynamicAccess.createInstanceFor[Throwable](
+          protoT.getClazzName,
+          List(classOf[String] → protoT.getMessage)).get
```

Comment:
That's a very good catch, @johanandren 
Do you think it would be alright if we first did these checks before creating the instance:
1. `dynamicAccess.getClassFor`
2. check that the class is a Throwable
3. check that there is a configured serializer for the class, using `Serialization.serializerFor`